packages/ui/Kbar.tsx

```diff
@@ -1,19 +1,19 @@
 import {
-  KBarProvider,
+  KBarAnimator,
   KBarPortal,
   KBarPositioner,
-  KBarAnimator,
-  KBarSearch,
+  KBarProvider,
   KBarResults,
-  useMatches,
+  KBarSearch,
   useKBar,
+  useMatches,
 } from "kbar";
 import { useRouter } from "next/router";
 
 import { useLocale } from "@calcom/lib/hooks/useLocale";
 import { isMac } from "@calcom/lib/isMac";
-import { Icon } from "@calcom/ui/Icon";
-import Tooltip from "@calcom/ui/v2/core/Tooltip";
+
+import { Icon, Tooltip } from ".";
```

Comment:
IMO all `calcom/ui` imports inside `calcom/ui` should be relative. If we're finding too many relative imports here is a mostly a symptom that there are many components that don't belong here. For example `feature` specific components should go into `calcom/features` instead.